CourseGrab/AppDelegate.swift

```diff
@@ -6,15 +6,17 @@
 //  Copyright © 2020 Cornell AppDev. All rights reserved.
 //
 
+import GoogleSignIn
 import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-
+    private let clientID = "894460636145-mulqctlklp1alf73u43kbdm5n7vulqga.apps.googleusercontent.com"
```

Comment:
we should definitely keep this secret. lets do secrets with a `Secrets/Keys.plist` and add that file to the `.gitignore`. if you want a reference you should take a look at `eatery-ios`